src/Bridge/Elasticsearch/DataProvider/Paginator.php

```diff
@@ -102,6 +104,8 @@ public function getItemsPerPage(): float
      */
     public function getIterator(): \Traversable
     {
+        $denormalizationContext = array_merge([AbstractNormalizer::ALLOW_EXTRA_ATTRIBUTES => true], $this->denormalizationContext);
```

Comment:
is the variable really needed?

Reply:
Yes, it prevents performing the same `array_merge` statement inside the `foreach` loop.